src/test/java/com/powsybl/openloadflow/ac/NonImpedantBranchWithBreakerIssueTest.java

```diff
@@ -50,8 +50,8 @@ void busBreakerAndNonImpedantBranchIssue() {
             assertEquals(400, bus.getV(), 0);
             assertEquals(0, bus.getAngle(), 0);
         }
-        assertEquals(-100, network.getGenerator("G1").getTerminal().getQ(), 0);
```

Comment:
@geofjamg here you have a relevant test that must be unchanged by your PR. We have at loading two voltage controls, both local. At equation system creation, we have to "merge" this voltage controls in order to create a remote voltage control with a distrib_q equation. If we don't do that, as you see, just one generator takes all the reactive power and reaches its limits too fast and the other one stay always PV. It is the issue on metrix test cases for instance.